pkg/config/apm.go

```diff
@@ -71,6 +71,8 @@ func setupAPM(config Config) {
 	config.BindEnv("apm_config.max_traces_per_second", "DD_APM_MAX_TPS", "DD_MAX_TPS")
 	config.BindEnv("apm_config.errors_per_second", "DD_APM_ERROR_TPS")
 	config.BindEnv("apm_config.disable_rare_sampler", "DD_APM_DISABLE_RARE_SAMPLER")
+	config.BindEnv("apm_config.remote_max_traces_per_second", "DD_APM_REMOTE_MAX_TPS")
```

Comment:
Not a blocker but it might be a touch more natural of a variable name if it was `apm_config.max_remote_traces_per_second` / `DD_APM_MAX_REMOTE_TPS`

Reply:
I'm not sure in this case as trace-agent might have other configs on remote config over time. Waiting for @gbbr to be back and check what he prefers

Thanks for the review!

Reply:
I think I agree here with switching the words "remote" and "max" to:
* DD_APM_MAX_REMOTE_TPS
* max_remote_traces_per_second
* etc...